samples/docker/docker-compose.yaml

```diff
@@ -12,61 +12,52 @@ services:
       SqlServer__Initialize: "true"
       BlobStore__ConnectionString: "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;"
       DataStore: "SqlServer"
-    ports:     
+    ports:
       - "8080:8080"
     depends_on:
       - sql
       - azurite
+
   azurite:
     image: mcr.microsoft.com/azure-storage/azurite
     restart: always
-    # # uncomment if you want to expose azure storage explorer against localhost
+    # Uncomment if you want to expose azure storage explorer against localhost
     # ports:
     #   - "10001:10001"
     #   - "10000:10000"
-  #Currently using azurite v2 for table storage as azurite v3 only supports blob and queue storage and v2 supports table storage. Once v3 supports queue storage this can be removed.
-  tableazurite:
-    build:
-      context: ../Azurite
-      dockerfile: ./Dockerfile
-    restart: always
-    environment: 
-      executable: table
-    # # uncomment if you want to expose azure storage explorer against localhost
-    # ports:
     #   - "10002:10002"
+
   sql:
-    build: 
+    build:
       context: ./../../
-      dockerfile: ./build/docker/sql/Dockerfile
+      dockerfile: ./docker/sql/Dockerfile
     restart: always
     environment:
       SA_PASSWORD: ${SAPASSWORD}
       ACCEPT_EULA: "Y"
-    # # uncomment if you want to expose sql management studio against localhost
```

Comment:
># uncomment [](http://example.com/codeflow?start=6&length=11)

Normally I leave double comment to signal that it should stay commented if you "uncomment" it

Reply:
Oh ok. I'll add them back